dfsclient/src/main/java/org/hpccsystems/dfs/client/RowServiceOutputStream.java

```diff
@@ -32,7 +32,7 @@
 public class RowServiceOutputStream extends OutputStream
 {
     private static final Logger  log                           = LogManager.getLogger(RowServiceOutputStream.class);
-    public static final int           DEFAULT_CONNECT_TIMEOUT_MILIS = 1000;                                              // 1 second connection timeout
+    public static final int      DEFAULT_CONNECT_TIMEOUT_MILIS = 5000; // 5 second connection timeout
```

Comment:
Just curious how we arrived at this default value. Default timeout values are always difficult to get right and there always has to be compromises. In this case, how often would we encounter this timeout on a single request? once per req, or perhaps once per req?

Reply:
Good question. I don't know if this is the best timeout, but I set it to match what we had RowServiceInputStream. When I am on VPN testing there are times where the original 1 sec timeout happened a lot, so I thought it would be good to change that default. 5s seems to work better for the VPN case, but it can be set by the user after that PR that Drea submitted.

Reply:
Yes, when I'm using this class I almost always override it with some constant or other, so I don't think it's a big issue.